undefined

```diff
@@ -1,6 +1,4 @@
-console.log('Hello World');

```

Comment:
The HTML declaration should not be mixed with the JavaScript code. Remove the <!DOCTYPE html> line from this JavaScript file.